fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/HashRdfContext.java

```diff
@@ -57,12 +57,12 @@
      * @throws javax.jcr.RepositoryException if repository exception occurred
      */
     public HashRdfContext(final FedoraResource resource,
-                          final IdentifierConverter<Resource, FedoraResource> idTranslator)
+                          final Converter<Resource, String> idTranslator)
             throws RepositoryException {
         super(resource, idTranslator);
 
         concat(getNodeStream(resource)
-                .flatMap(n -> nodeConverter.convert(n).getTriples(idTranslator, PROPERTIES))
+                .flatMap(n -> nodeConverter.apply(n).getTriples(idTranslator, PROPERTIES))
```

Comment:
I'm not clear on what `nodeConverter` is here. There's no field named that here or in `NodeRdfContext`? (But there's a `nodeConverter()` method...?)